tests/stubs.cpp

```diff
@@ -30,3 +30,10 @@ Config *control = nullptr;
 // negatively affect test harness.
 void GFX_ShowMsg(const char *, ...) {}
 void DEBUG_ShowMsg(const char *, ...) {}
+
+void DEBUG_HeavyWriteLogInstruction(){}
+void MSG_Add(const char *, const char *){};
+const char *MSG_Get(char const *)
+{
+	return nullptr;
+};
```

Comment:
Can you add a carriage return at the end? 

GitHub flags these with the little red circle - 
![2021-05-26_15-22](https://user-images.githubusercontent.com/1557255/119738802-21d43980-be36-11eb-9f5d-b2de15526857.png)

Reply:
will do